packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -156,19 +155,23 @@ exports.questions = {
 		validators: [new RequiredValidator()]
 	}),
 	representationUpload: new MultiFileUploadQuestion({
-		title: 'Upload representations from other parties',
-		question: 'Upload the representations',
-		fieldName: 'representation-upload',
-		validators: [new RequiredValidator('You must add your documents')]
+		title: '**************placeholder*********************',
+		question: '**************placeholder*********************',
+		fieldName: 'representation-upload'
+	}),
+	reportUpload: new MultiFileUploadQuestion({
+		title: '**************placeholder*********************',
+		question: '**************report-upload-placeholder*********************',
+		fieldName: 'report-upload'
 	}),
-	planningOfficersReportUpload: new MultiFileUploadQuestion({
+	planningOfficersUpload: new MultiFileUploadQuestion({
```

Comment:
I think this section is the artifact of a merge? the name was changed my pr merged earlier to 'planningOfficersReportUpload'. Pr for reference: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3055